src/main/java/net/spy/memcached/protocol/TCPMemcachedNodeImpl.java

```diff
@@ -88,6 +88,8 @@ public abstract class TCPMemcachedNodeImpl extends SpyObject implements
   private volatile long lastReadTimestamp = System.nanoTime();
   private MemcachedConnection connection;
   private TLSConnectionHandler tlsConnectionHandler;
+  private SSLContext sslContext;
```

Comment:
You don't need to track this because you already have the `connectionFactory` object tracked in this class